cxplat/inc/cxplat.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+#pragma once
+
+#include "cxplat/fault_injection.h"
```

Comment:
My personal preference will be to not folder path to specify the name, but instead prefix with `cxplat_`:
```suggestion
#include "cxplat_fault_injection.h"
```
It's not unheard of working on a similar file name "memory.h" at multiple levels, and it's confusing in the editor to see multiple memory.h files.

Reply:
Done